Document Keyboard Shortcuts (#135)
mixxxdj/manual: source/chapters/library.rst

**Loading tracks**
  To load a track into a :ref:`deck <interface-decks>`, you can either drag it

>   to the waveform display, use the context menu or use shortcut keys. Go to the chapter

Reviewer: ```suggestion
  to the waveform display, use the context menu, or use shortcut keys. Go to the chapter
```

>   to the waveform display, use the context menu, or use shortcut keys. Go to the chapter

Reviewer 2: ```suggestion
  to the waveform display, use the context menu or use :term:`keyboard shortcuts <shortcut>`. Go to the chapter
```

  load a track into a deck.
* By :ref:`control-keyboard` to load the selected track in the library track
  table.
<<<<<<< HEAD
* Using keyboard shortcut keys to load track in Deck 1 with :kbd:`Shift` + :kbd:`Left` and Deck 2
 with :kbd:`Shift` + :kbd:`Right`
=======
* Using keyboard shortcut keys to load track in Deck1 with Shift + Left and Deck2
 with Shift + Right

> >>>>>>> b1eb71d86fc5670f886799a713ad47df7f7a4a08

Reviewer 2: Merge conflict marker.

  load a track into a deck.
* By :ref:`control-keyboard` to load the selected track in the library track
  table.
* Using keyboard :term:`shortcut <shortcuts>` keys to load track in Deck 1 with :kbd:`Shift` + :kbd:`Left` and Deck 2

>  with :kbd:`Shift` + :kbd:`Right`

Author: Ok!

>  with :kbd:`Shift` + :kbd:`Right`

Author: I don't know if i did this right, please review @ronso0

  example, on macOS, dragging-and-dropping a track from iTunes onto one of
  Mixxx's waveform displays will load it into that deck.

> unLoading Tracks

Reviewer: ```suggestion
Unloading Tracks
```

==============

> 

Reviewer: If we're adding a section for this, the eject buttons in the decks should also be mentioned.

Author: Thank you for the guide!

  example, on macOS, dragging-and-dropping a track from iTunes onto one of
  Mixxx's waveform displays will load it into that deck.

> UnLoading Tracks

Reviewer 3: ```suggestion
Unloading Tracks
```

  example, on macOS, dragging-and-dropping a track from iTunes onto one of
  Mixxx's waveform displays will load it into that deck.

> 
> Unloading Tracks
> 

Reviewer 3: Your previous additions are missing
```suggestion

Unloading Tracks
* To unload tracks use the keyboard shortcuts :kbd:`Shift` + :kbd:`Alt` + :kbd:`Left` for Deck 1 and :kbd:`Shift` + :kbd:`Alt` + :kbd:`Right` for Deck 2.

* You can also use the eject track button to unload track from a deck. Look for a button with this icon:

  .. figure:: ../_static/Mixxx-eject-button.png
     :align: center
     :width: 100%
     :figwidth: 100%
     :alt: Mixxx library - Eject Button
     :figclass: pretty-figures
```

Author: You asked me to remove the picture

Reviewer 3: sorry, I'm confused some reviews.
I trimmed the suggestion above. please check.

We can add the picture later on, so this get's merged soonish.

Also I went through previous reviews so we don't miss anything and don't have contradicting change proposal.
Sorry for the hazzle!

  Mixxx's waveform displays will load it into that deck.

Unloading Tracks
==============

Tracks can be unloaded into a deck in several ways:

* To unload track in Deck 1, use the keyboard shortcut Shift + Left. In Deck 2 you can
  unload track with Shift + Right

* You can use the eject track button to unload track from a deck. This button is

>   located at the bottom right, below the waveform.

Author: I would implement this now @ronso0  . Thank you so much for taking out time to review this.

>   located at the bottom right, below the waveform.

Author: Hello @ronso0 I just implemented this, but i don't really know how to implement this `QT_SCREEN_SCALE_FACTORS=2` today is the first time I am using RST actively. The first time I used it involved no addition of pictures. What do you think about the coloured eject button? since you wanted more emphasis on it

>   located at the bottom right, below the waveform.

Reviewer 3: oh sorry, forgot you're ne wto mixxx :)
scale factors refers to running Mixxx with an user-specified scaled GUI, for example on 4k screens.
Instructions are here:
https://github.com/mixxxdj/mixxx/wiki/troubleshooting#graphical-user-interface-gui-is-too-big-or-too-small
If you don't succeed, let me know. I can upload the picture myself later today.
> What do you think about the coloured eject button?

either blur the sorroundungs or desaturate everything except the button and use a colored icon.

>   located at the bottom right, below the waveform.

Author: > oh sorry, forgot you're ne wto mixxx :)
> scale factors refers to running Mixxx with an user-specified scaled GUI, for example on 4k screens.
> Instructions are here:
> https://github.com/mixxxdj/mixxx/wiki/troubleshooting#graphical-user-interface-gui-is-too-big-or-too-small
> If you don't succeed, let me know. I can upload the picture myself later today.
> 
> > What do you think about the coloured eject button?
> 
> either blur the surroundings or desaturate everything except the button and use a colored icon.

Hello @ronso0 I read the [instructions](https://github.com/mixxxdj/mixxx/wiki/troubleshooting#graphical-user-interface-gui-is-too-big-or-too-small) and can't seem to understand how to implement the scale factor. The instruction explains why an image maybe too big or small and how to remove scale factor from your device. can you please do it so I can learn from what you did? Thank you!

>   located at the bottom right, below the waveform.

Reviewer 3: This is what I had in mind.
![Mixxx-eject-button png](https://user-images.githubusercontent.com/5934199/95598055-bdec3480-0a4f-11eb-9b06-2dcc4762bf8a.png)
or this
![Mixxx-eject-button png_blur](https://user-images.githubusercontent.com/5934199/95598573-5f738600-0a50-11eb-90e6-63bf23dbfcab.png)

We may not need the entire deck, so feel free to crop or tweak the image further, as you like. Check what looks best in the manual context.
Then you may `git add [path to image] && git commit --amend` so it's part of your previous commit "Adding image of eject button".
`git push -f` afterwards.

>   located at the bottom right, below the waveform.

Author: Hello @ronso0 the first Image is so nice! I would work on this now. Hoping to learn more from you! I really like the first image.

>   located at the bottom right, below the waveform.

Author: Hello @ronso0 I just implemented this now! What do you think?

>   located at the bottom right, below the waveform.

Author: @ronso0 I ran ```git commit -m```  before ```git --amend -m```. It was a mistake, am sorry

>   located at the bottom right, below the waveform.

Reviewer 2: Note that this PR goes into the `manual-2.2.x` branch, so the Screenshots should match the default skin used in Mixxx 2.2. We'll need to update the Screenshots later for Mixxx 2.3.

>   located at the bottom right, below the waveform.

Reviewer 3: okay, so let's leave the screenshot out of this
@hannydevelop unless you're willing to install Mixxx 2.2.4 and prepare a screenshot with the Deere skin similiar to what I posted above.
but we can also add that after #154 has been merged.
As you like..

>   located at the bottom right, below the waveform.

Author: > okay, so let's leave the screenshot out of this
> @hannydevelop unless you're willing to install Mixxx 2.2.4 and prepare a screenshot with the Deere skin similiar to what I posted above.
> but we can also add that after #154 has been merged.
> As you like..

Ok, I would remove the Image then

>   located at the bottom right, below the waveform.

Author: > > okay, so let's leave the screenshot out of this
> > @hannydevelop unless you're willing to install Mixxx 2.2.4 and prepare a screenshot with the Deere skin similiar to what I posted above.
> > but we can also add that after #154 has been merged.
> > As you like..
> 
> Ok, I would remove the Image then

Image has been removed

>   located at the bottom right, below the waveform.

Reviewer 3: please also remove the lonely `Unloading Tracks` header.

>   located at the bottom right, below the waveform.

Reviewer 3: Updated by current review

>   located at the bottom right, below the waveform.

Reviewer 3: Eject button image can be added later on, keyboard should be mentioned

  example, on macOS, dragging-and-dropping a track from iTunes onto one of
  Mixxx's waveform displays will load it into that deck.

<<<<<<< HEAD
Unloading Tracks
================

* To unload track in Deck 1, use the keyboard shortcut :kbd:`Shift` + :kbd:`Left`. In Deck 2 you can
  unload track with :kbd:`Shift` + :kbd:`Right`

* You can also use the eject track button to unload track from a deck. 
=======
UnLoading Tracks
================

* To unload track in Deck 1, use the keyboard shortcut Shift + Left. In Deck 2 you can
  unload track with Shift + Right

* You can also use the eject track button to unload track from a deck. Look for a button with this icon:

  .. figure:: ../_static/Mixxx-eject-button.png
     :align: center
     :width: 100%
     :figwidth: 100%
     :alt: Mixxx library - Eject Button
     :figclass: pretty-figures

     Mixxx library - Eject Button

> >>>>>>> b1eb71d86fc5670f886799a713ad47df7f7a4a08

Reviewer 2: Merge conflict marker

  library. Use the :kbd:`ARROW UP` and :kbd:`ARROW DOWN` keys to scroll in the
  list of results.

> .. note:: note:: If the search input field has focus, the Mixxx keyboard shortcuts are

Reviewer 3: please remove redundant `note::` here

Author: ooh! Done! Please review @ronso0

  library. Use the :kbd:`ARROW UP` and :kbd:`ARROW DOWN` keys to scroll in the
  list of results.

> .. note:: If the search input field has focus, the Mixxx keyboard :term:`shortcuts <shortcuts>` are

Reviewer 3: This now feels redundant: I think the shortcuts reference is obsolete here when we have "see :ref:`control-keyboard`", so please remove the change.

I'm sorry this gets confusing with multiple reviewers...

Author: Done!
